test/features/data_usability_rating.feature

```diff
@@ -1,55 +1,67 @@
 @data_usability_rating
 Feature: Data usability rating
 
-    Scenario: As an admin user of my organisation, when I create a dataset with a HTML resource, I can verify the score is 0
+    Scenario: As an organisation admin, when I create a dataset with an open license and HTML resource, I can verify the score is 0
         Given "TestOrgAdmin" as the persona
         When I log in
         And I create a dataset with license "other-open" and "HTML" resource file "html_resource.html"
         Then I wait for 10 seconds
         When I reload
-        Then I should see an element with xpath "//h2[contains(text(), 'Openness')]"
+        Then I should see data usability rating 0
+        When I click the link with text that contains "Test resource"
+        Then I should see data usability rating 0
```

Comment:
If I'm understanding this correctly, This will check the usability rating on the dataset page and then it will go to the resource page and do the same check. But I cannot see usability rating displayed on resource pages.

Reply:
It's there, at the bottom of the page, after the QA process has run.

Reply:
Thanks, caught out by looking at dev which doesn't have it on the resource. Test, Staging and Prod do.